habitat_baselines/config/default.py

```diff
@@ -17,48 +17,87 @@
 # Config definition
 # -----------------------------------------------------------------------------
 _C = CN()
-_C.SEED = 100
+_C.BASE_TASK_CONFIG_PATH = "configs/tasks/pointnav.yaml"
+_C.TASK_CONFIG = get_config()  # task_config will be stored as a config node
```

Comment:
Task config will be overwritten with default config otherwise:
```suggestion
_C.TASK_CONFIG = CN()  # task_config will be stored as a config node
```

Reply:
I agree, this makes the behavior more obvious for user. But this does prohibit user from overwrite task_config in `ppo.yaml`